app/budget/services/budgetSelectors.js

```diff
@@ -264,11 +266,12 @@ budgetApp.service('budgetSelectors', function () {
 				// Set 'actual instructor cost'
 				// When sectionGroup.instructorCost is overridden, we will display this value
 				sectionGroupCost.actualInstructorCost = instructorCostSubTotal;
+				sectionGroupCost.actualInstructorCostDisplay = toCurrency(sectionGroupCost.actualInstructorCost);
 
 				// Set totalCost
 				var totalCost = supportCostSubTotal + instructorCostSubTotal;
 				sectionGroupCost.totalCost = parseFloat(totalCost).toFixed(2);
-
+				sectionGroupCost.totalCostDisplay = toCurrency(sectionGroupCost.totalCost);
```

Comment:
Does the currency version belong as a property on the model, or is it a view concern?